.mocharc.json

```diff
@@ -0,0 +1,6 @@
+{
+  "require": "ts-node/register",
+  "spec": "packages/ecdh/tests/**/*.test.ts",
+  "timeout": 20000,
+  "recursive": true
```

Comment:
```suggestion
  "recursive": true,
  "exit": true
```

Reply:
@YashBit please don't mark conversation as resolved until we agreed on a solution.  
You don't have to follow my suggestion. You may have a good reason to ignore it: in this case please share it so we can discuss